.travis.yml

```diff
@@ -1,16 +1,26 @@
+dist: bionic
 language: rust
 cache: cargo
 rust: nightly
+sudo: true
+
 before_script:
+  - sudo add-apt-repository -y ppa:deadsnakes/ppa
+  - sudo apt-get update
+  - sudo apt-get install python3.7 python3.7-distutils
+  - curl https://bootstrap.pypa.io/get-pip.py | sudo python3.7
+  - rustup toolchain install nightly-2019-07-25
+  - rustup default nightly-2019-07-25
   - rustup component add rustfmt-preview
   - rustup update
   - cargo update
+  - sudo pip3.7 install -r ./tests/requirements.txt
 
 script:
   - cargo fmt --all -- --check
   - cargo build --all
   - cargo test --all
-  - ./tests/run.sh
+  - sudo "PATH=$PATH" ./tests/run.sh
```

Comment:
```suggestion
  - sudo --preserve-env=PATH ./tests/run.sh
```

Reply:
Yeah, I wanted to do that in the first place, but travis' sudo doesn't allow it (see build [#371](https://travis-ci.org/raven-os/nest/builds/565551264))